.travis.yml

```diff
@@ -47,6 +47,29 @@ matrix:
            PROJECT_URL=https://github.com/joblib/joblib.git
       python: 3.7
       if: commit_message =~ /(\[ci downstream\]|\[ci joblib\])/
+    - os: linux
+      env: PROJECT=ray TEST_REQUIREMENTS="setproctitle psutil"
```

Comment:
Consider putting these directly into the `pip install` below. It was only necessary when factoring out code.